### PR TITLE
PLAT-24356: Allow setting updatedAt field when running alpha script

### PR DESCRIPTION
**Pull request by yossipapi**

### CODE REVIEW

**Review by inbal-ben-david — APPROVED** [review]

**Comment by hilak** [comment]
@yossipapi  @inbal-ben-david  I've piggybacked on this PR to add the alpha script which updates the entry property.
Also @yossipapi  we've talked about changing the static property name on entry, but the setter name didn't match, I've updated it

File: alpha/scripts/utils/setEntryUpdateDate.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+KalturaStatement::setDryRun($dryRun);
+
+$entryMappings = file($mapping, FILE_IGNORE_NEW_LINES);
+
+$counter = 0;
+foreach ($entryMappings as $entryMapping)
+{
+	list ($entryId, $updatedAt) = explode(",", $entryMapping);
+    $updatedAt = $isDateString ? strtotime($updatedAt) : $updatedAt;
```
Comment (inbal-ben-david) [inline alpha/scripts/utils/setEntryUpdateDate.php]: indentation lines 23 and 25

File: alpha/scripts/utils/setEntryUpdateDate.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    entry::setAllowOverrideReadOnlyFields(true);
```
Comment (inbal-ben-david) [inline alpha/scripts/utils/setEntryUpdateDate.php]: why need to do it for each entry and not once?

**Comment by hilak** [comment]
@inbal-ben-david  addressed.

**Review by inbal-ben-david — APPROVED** [review]

**Pull request merged**